features/step_definitions/edit_review_steps.rb

```diff
@@ -0,0 +1,50 @@
+When(/^I attempt to edit any review$/)do
+  new_user = User.new
+  new_user.email = "user1@tamu.edu"
+  new_user.password_digest = "12345"
+  new_user.save
+  current_user = User.where(:email => 'user1@tamu.edu').first
```

Comment:
I'm just raising my previous concern that this doesn't have the same effect as logging in, but I don't know how the login system works that well.
Need @dfields16 input.